admin/ajax.php

```diff
@@ -388,3 +371,17 @@ function wpseo_kill_blocking_files() {
 
 	wpseo_ajax_json_echo_die( '' );
 }
+
+/**
+ * Handles the posting of a new FB admin.
+ *
+ * @deprecated 7.1
+ * @codeCoverageIgnore
```

Comment:
The rest of the plugin always uses // @codeCoverageIgnoreStart and // @codeCoverageIgnoreEnd blocks instead of @codeCoverageIgnore for each function. Suggested to a block here as well to make code more DRY and consistent.

Reply:
That's not true, I copied this from another file actually.